tasklist/client/src/Tasks/Task/Details/TabListNav.tsx

```diff
@@ -27,17 +28,24 @@ const TabListNav: React.FC<Props> = ({className, label, items}) => {
   return (
     <nav className={cn(className, styles.tabs, 'cds--tabs')}>
       <div className="cds--tab--list" aria-label={label}>
-        {items.map(({key, title, label, selected, href}) => (
+        {items.map(({key, title, label, selected, href, visible}) => (
           <button
             key={key}
             type="button"
             role="link"
             aria-label={label}
             aria-current={selected ? 'page' : undefined}
-            className={cn('cds--tabs__nav-item', 'cds--tabs__nav-link', {
-              ['cds--tabs__nav-item--selected']: selected,
-            })}
+            className={cn(
+              {[styles.hidden]: visible === false},
+              'cds--tabs__nav-item',
+              'cds--tabs__nav-link',
+              {
+                ['cds--tabs__nav-item--selected']: selected,
+              },
+            )}
             tabIndex={selected ? 0 : -1}
+            hidden={visible === false}
+            aria-hidden={visible === false}
```

Comment:
Can be slightly improve if you create a var like `const isVisible visible ===false`